app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,12 @@
+- provide(:title, "Log in")
```

Comment:
It does, but we're going to ditch it because we just need a developer login at the moment to test admin vs non-admin functionality.